### PR TITLE
Fixes bug 1148566 - Concat serverStars to stars

### PR DESCRIPTION
**Pull request by bugzPDX**
@peterbe serverStars were being ignored.

### CODE REVIEW

File: airmozilla/starred/static/starred/js/star_event.js [hunk 1]
```diff
@@ -70,6 +70,8 @@ var Stars = (function() {
                 // jquery array comparison http://stackoverflow.com/a/7726509/205832
                 if (!($(stars).not(serverStars).length === 0 &&
                       $(serverStars).not(stars).length === 0)) {
+                    stars = stars.concat(serverStars);
```
Comment (peterbe) [inline airmozilla/starred/static/starred/js/star_event.js]: This makes me nervous. What if `stars` is not empty or if there is an item in both arrays. Then you'd get repeated IDs in there. 
Why not do something like this (untested):
```javascript
serverStars.forEach(function(id) {
    if (stars.indexOf(id) === -1) { 
        stars.push(id); /// Should that be unshift() instead??!?
    }
});
```

File: airmozilla/starred/templates/starred/events.html [hunk 2]
```diff
@@ -1,2 +1,6 @@
-{% include 'main/_events.html' %}
-{% include 'main/_paginate.html' %}
+{% if events %}
+    {% include 'main/_events.html' %}
+    {% include 'main/_paginate.html' %}
+{% else %}
+    You have no starred events. Yet.
```
Comment (peterbe) [inline airmozilla/starred/templates/starred/events.html]: Any text should always be in a tag. E.g. `<p>You have...`

File: airmozilla/starred/static/starred/js/star_event.js (hunk 1 above)
Comment (bugzPDX) [inline airmozilla/starred/static/starred/js/star_event.js]: Yes they could get repeated but the server is going to sort it all out. I think changing it is just going to slow everything down for not a lot of benefit.

Comment (peterbe) [inline airmozilla/starred/static/starred/js/star_event.js]: A ha! I get it. The `stars` gets set to `stars = response.ids;` a couple of milliseconds later.

**Comment by peterbe** [comment]
@bugzPDX Oops! I merged this too quickly. Would you mind addressing the comment I made about the "You have no starred events. Yet" text being in a `<p>` tag?

**Pull request merged**